src/main/java/frc/robot/pneumatics/PnematicsConstants.java

```diff
@@ -0,0 +1,12 @@
+package frc.robot.pneumatics;
+
+public class PnematicsConstants {
+
+  // We are using arrays to store the forward and reverse channels of each solenoid //
+  // Allocate less variables and looks cleaner //
```

Comment:
"fewer", not "less"

Also, the comments need info on how yo determine the ports for the left and right solenoids.

Reply:
They are in the slides on how to do so